proposals/xxxx-continous-integration.md

```diff
@@ -0,0 +1,54 @@
+# Configuration management
+
+* Proposal: [SDL-xxxx](xxxx-Configuration-Management.md)
+* Author: [Vadym Kushnirenko](https://github.com/vkushnirenko-luxoft)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core/??]
+
+## Introduction
+
+This proposal is about configuration management system for continuous building and testing code.
+
+## Motivation
+
+The development process should be open and transparent. Any person could deploy it on his own side, use it and propose recomendations about improvements.
```

Comment:
@vkushnirenko-luxoft 
- I think better is "_Any person should be able to deploy it on his side_ ..." 
- _recommendation_